feg/gateway/services/n7_n40_proxy/n7_n40_proxy/main.go

```diff
@@ -37,21 +39,37 @@ func main() {
 		glog.Fatalf("Error creating N7_N40 Proxy service: %s", err)
 	}
 
+	n7config, err := n7.GetN7Config()
+	if err != nil {
+		glog.Fatalf("Error fetching config: %s", err)
+	}
 	cloudReg := registry.Get()
 	dbClient, err := policydb.NewRedisPolicyDBClient(cloudReg)
 	if err != nil {
 		glog.Fatalf("Error connecting to redis store from N7_N40 Proxy: %s", err)
 	}
-
-	sessController, err := servicers.NewCentralSessionController(dbClient)
+	policyClient, err := n7.NewN7Client(&n7config.Server)
+	if err != nil {
+		glog.Fatalf("Creating N7 Client failed: %s", err)
+	}
+	sessController, err := servicers.NewCentralSessionController(n7config, dbClient, policyClient)
 	if err != nil {
 		glog.Fatalf("Error creating session controller in N7_N40 Proxy: %s", err)
 	}
+	notifHandler := notify.NewNotificationHandler(&n7config.Client, cloudReg)
+	if notifHandler == nil {
+		glog.Fatalf("Error creating noitification handler")
+	}
```

Comment:
I think handlers shouldn't be added in main 

Check other systems how this works: see how clean is the s8_proxy main [here](https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Amagma%2Fmagma.git&branch=master&file=feg%2Fgateway%2Fservices%2Fs8_proxy%2Fs8_proxy%2Fmain.go&editor=JetBrains&version=v1.2.1&utm_product_name=IntelliJ+IDEA&utm_product_version=2021.2.2&start_row=30&start_col=5&end_row=30&end_col=5). Note session_proxy main is a little mess. Lets try no to repeat the same 

See how the hanlders are added on s8_proxy [here](https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Amagma%2Fmagma.git&branch=master&file=feg%2Fgateway%2Fservices%2Fs8_proxy%2Fservicers%2Fgtp_handlers.go&editor=JetBrains&version=v1.2.1&utm_product_name=IntelliJ+IDEA&utm_product_version=2021.2.2&start_row=25&start_col=10&end_row=25&end_col=10)